File: codingdojo.org/Fizzbuzz.java
for (int i = 1; i <= 100; i++) {
    if (i % (3 * 5 ) == 0) {
        if (i % 3 == 0) System.out.print("Fizz");
        if (i % 5 == 0) System.out.print("Buzz");
        System.out.println("");
    } else {
        System.out.println(i);
    }
}

File: codingdojo.org/fizzbuzz.hs
fizzbuzz :: [String]
fizzbuzz = map fb [1..100]
  where fb i
          | i `mod` (3 * 5) == 0 = "FizzBuzz"
          | i `mod` 3 == 0 = "Fizz"
          | i `mod` 5 == 0 = "Buzz"
          | otherwise = show i
